viewers/wicket/ui/src/main/java/org/apache/isis/viewer/wicket/ui/components/scalars/ScalarPanelTextFieldAbstract.html

```diff
@@ -86,7 +86,7 @@
     </wicket:fragment>
 
     <wicket:fragment wicket:id="textareaInlinePrompt">
-        <textarea class="form-control input-sm" wicket:id="scalarValue"></textarea>
+        <textarea class="form-control input-sm" wicket:id="scalarValue" readonly="readonly"></textarea>
```

Comment:
I believe this is valid, but unfortunately changes the appearance, which we yet have not found a way to fix without a headache considering there are different bootstrap themes a user can choose. I figure the CSS fix you are suggesting works for some themes, but unfortunately not for all of them.